### PR TITLE
chore: MySQL e2e UI tests

### PR DESCRIPTION
**Pull request by cduflo**
#### Summary

⚠️ **If you're contributing to a plugin please read this section of the [contribution guidelines](https://github.com/cloudquery/cloudquery/blob/main/CONTRIBUTING.md#open-core-vs-open-source) 🧑‍🎓 before submitting this PR** ⚠️

### CODE REVIEW

**Review by erezrokah — APPROVED** [review]

**Review by erezrokah — APPROVED** [review]

File: plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  pluginUrl: string;
+}
+
+export const login = async (page: Page) => {
+  await page.goto('https://cloud.cloudquery.io/auth/login');
+
+  const { email, password } = getMainTestUser();
+
+  await page.getByLabel('Email Address').click();
+  await page.getByLabel('Email Address').fill(email);
```
Comment (jackcloudquery) [inline plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts:26]: We could use the util methods which you copied into e2e-helpers but if it works without them consistently - I am okay with that

File: plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  const iframeElement = page.frame({ name: 'Plugin UI' });
+
+  if (!iframeElement) {
+    throw new Error('iframe not found');
+  }
+
+  await fillInput(iframeElement, '[name="displayName"]', pluginNewName);
+  await fillFieldsSteps?.(iframeElement);
+
+  await click(iframeElement, iframeElement.getByRole('button', { name: 'Test connection' }));
```
Comment (jackcloudquery) [inline plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts:69]: As I remember the label for this button is dynamic, so it probably should also be configurable

File: plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  test.setTimeout(300_000);
+  await page.goto(getRootUrl());
+
+  await expect(page.getByText(`Create a ${kind}`)).toBeVisible();
+
+  await page.locator('input[type="text"]').clear();
+  await page.locator('input[type="text"]').fill(pluginName);
+
+  await click(page, page.getByRole('button', { name: pluginLabel }));
+  await expect(page.getByText('Previewing')).toBeVisible({ timeout: 3000 });
```
Comment (jackcloudquery) [inline plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts:96]: What for? Isn't 5000 a default?

File: plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+  await expect(
+    iframeElement.getByRole('textbox', {
+      name: `${kind === 'destination' ? 'Destination' : 'Source'} name`,
+    }),
+  ).toHaveValue(pluginNewName);
+
+  await fillFieldsSteps?.(iframeElement);
+
+  await click(iframeElement, iframeElement.getByRole('button', { name: 'Test connection' }));
```
Comment (jackcloudquery) [inline plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts:119]: The same

**Review by jackcloudquery — CHANGES_REQUESTED** [review]

File: plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts (hunk 1 above)
Comment (cduflo) [reply plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts]: handled in the lib, thanks for the point out https://github.com/cloudquery/plugin-config-ui-lib/pull/51/files?w=1#diff-97a4a70134d628365b637c8f2f2c1ed7951322771a8b97559a13932ecaac8a90

File: plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts (hunk 3 above)
Comment (cduflo) [reply plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts]: TIL :)

**Review by jackcloudquery — APPROVED** [review]

**Pull request merged**